projects/storefrontlib/src/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, Input } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+/**
+ * This component renders form errors.
+ */
+@Component({
+  selector: 'cx-form-errors',
+  templateUrl: './form-errors.component.html',
```

Comment:
The component must be setup to use the OnPush ChangeDetection strategy.

Reply:
fixed